.github/workflows/test.yml

```diff
@@ -8,8 +8,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [ '2.7', '3.0', '3.1', '3.2', '3.3', jruby ]
```

Comment:
I removed this because `activerecord-jdbc-adapter` doesn't support Rails 7.1 yet 
https://github.com/jruby/activerecord-jdbc-adapter/issues/1146

Instead of this, I added to the matrix for JRuby + gemfiles/rails-7-0.gemfile.